### DISCUSSION TITLE
pcl还是会报毒

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#3132] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/42478088/e65f60f5-acab-49fb-b462-e393c278ea3a)

**Comment by RainbowTaro2022** [comment]
hmcl有18个是报毒的
![image](https://github.com/Hex-Dragon/PCL2/assets/42478088/07012e68-0282-40fb-840e-772ea5947185)

**Reply by zkitefly** [reply]
我检查的是绿的

![image](https://github.com/Hex-Dragon/PCL2/assets/64117916/a6971bf8-9d98-4dd8-9409-4ba147cec8b6)

**Reply by LTCatt** [reply]
@zkitefly 你扫描的是 URL……

**Reply by zkitefly** [reply]
> @zkitefly 你扫描的是 URL……

那个 url 是 ci 的啊，平台会自己下载然后检查

**Comment by MoYuan-CN** [comment, marked as answer]
第三方审查器报毒很正常
如果 PCL 被 Windows Defender 误删，请参考内置帮助库：启动器 > 将 PCL 添加到 Windows 安全中心白名单 或  #56 #666 解决
